src/plugins/embeddable/public/react_embeddable_system/react_embeddable_renderer.test.tsx

```diff
@@ -105,22 +104,4 @@ describe('react embeddable renderer', () => {
       )
     );
   });
-
-  it('registers the API with the parent API', async () => {
```

Comment:
This test is removed because the responsibility for adding the child API to the `children$` subject has been moved to the rendering component.